FullStorySegmentMiddleware/Classes/FullStoryMiddleware.m

```diff
@@ -124,6 +128,96 @@ - (SEGPayload *) getNewPayloadWithFSURL:(SEGContext * _Nonnull)context {
     return newPayload;
 }
 
+
+- (NSDictionary *) getSuffixedProps: (NSDictionary *)properties{
+    //TODO: Segment will crash and not allow props to have curcular dependency, but we should handle it here anyways
+    
+    NSMutableDictionary *props = [[NSMutableDictionary alloc] initWithCapacity:[properties count]];
+    for(id key in properties){
```

Comment:
nit: FS style has spaces: `for (id key in properties) {`

Reply:
nit: if we know the type of `key`, we should specify it (is `key` usually a `NSString *`)?